Makefile

```diff
@@ -32,31 +32,46 @@ endif
 MSBUILD_ARGS := /nologo '/consoleloggerparameters:Verbosity=minimal;summary' /p:Configuration=$(BUILD_CONFIGURATION)
 
 ifneq ($(BUILD_LOG_PATH),)
-	MSBUILD_ARGS += /filelogger '/fileloggerparameters:Verbosity=normal;logFile=$(BUILD_LOG_PATH)'
+    MSBUILD_ARGS += /filelogger '/fileloggerparameters:Verbosity=normal;logFile=$(BUILD_LOG_PATH)'
 endif
 
+MSBUILD_MAIN_ARGS := $(MSBUILD_ARGS)
+MSBUILD_BOOTSTRAP_ARGS := $(MSBUILD_ARGS)
```

Comment:
I went to split this into MSBUILD_ARGS and DOTNET_ARGS, but then I found that the `-r` flag is [just a simple passthrough to `/p:RuntimeIdentifier=...`](https://github.com/dotnet/cli/blob/5c67b11d62a353eaf1cf95b2bd507d1c2bd15183/src/dotnet/CommonOptions.cs#L46). I changed it to use that, because we have similar things with `/p:RoslynRuntimeIdentifier`, and it's less confusing to have them be unified syntax.